src/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -432,4 +432,7 @@
   <data name="net_context_buffer_too_small" xml:space="preserve">
     <value>Insufficient buffer space. Required: {0} Actual: {1}.</value>
   </data>
+  <data name="net_nego_channel_binding_not_supported" xml:space="preserve">
+    <value>No support for channel binding on non-Windows</value>
```

Comment:
Please change to "No support for channel binding on operating systems other than Windows."